src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -383,6 +384,6 @@ public void onLoad(Run<?, ?> run) {
 
 	@Override
 	public Collection<? extends Action> getProjectActions() {
-		return jacocoProjectActions;
+		return jacocoProjectAction != null ? Arrays.asList(jacocoProjectAction) : Collections.emptyList();
```

Comment:
I would use Collections.singletonList() here instead of Arrays.asList()

Reply:
ok

Reply:
done